src/Client/LocalConnection.h

```diff
@@ -50,19 +52,28 @@ struct LocalQueryState
     Progress progress;
     /// Time after the last check to stop the request and send the progress.
     Stopwatch after_send_progress;
+    Stopwatch after_send_profile_events;
+
+    std::unique_ptr<CurrentThread::QueryScope> query_scope_holder;
 };
 
 
 class LocalConnection : public IServerConnection, WithContext
 {
 public:
-    explicit LocalConnection(ContextPtr context_, bool send_progress_ = false);
+    explicit LocalConnection(
+        ContextPtr context_, bool send_progress_ = false, bool send_profile_events_ = false, const String & server_display_name_ = "");
```

Comment:
Better to move `server_display_name` after context and without default parameter. Actually let's remove all default parameters here (for `send_progress` and `send_profile_events`).

Reply:
@kssenii  I put server_display_name_  in last position and set default value for it because of 
![image](https://user-images.githubusercontent.com/8181003/157015040-44b387f5-a4f7-4b9b-a066-a5f0c3c1db4e.png)